mslib/mscolab/utils.py

```diff
@@ -55,8 +59,20 @@ def get_message_dict(message):
     }
 
 
+def os_fs_create_dir(dir):
```

Comment:
Mmh. I think we need to go to an fs-only approach in combination with validating/converting any path communicated from an external source (QT-dialogues, JSON files, etc.).

Good start in doing that.